Gems/Atom/RHI/Code/Source/RHI/RHISystem.cpp

```diff
@@ -96,92 +97,113 @@ namespace AZ
             if (physicalDevices.empty())
             {
                 AZ_Printf("RHISystem", "Unable to initialize RHI! No supported physical device found.\n");
-                return nullptr;
+                return;
             }
 
-            AZStd::string preferredUserAdapterName = RHI::GetCommandLineValue("forceAdapter");
-            AZStd::to_lower(preferredUserAdapterName.begin(), preferredUserAdapterName.end());
-            bool findPreferredUserDevice = preferredUserAdapterName.size() > 0;
+            static const char* MultiGPUCommandLineOption = "rhi-multiple-devices";
+            AZStd::string multipleDevicesValue = RHI::GetCommandLineValue(MultiGPUCommandLineOption);
+
+            RHI::PhysicalDeviceList usePhysicalDevices;
 
-            RHI::PhysicalDevice* preferredUserDevice{};
-            RHI::PhysicalDevice* preferredVendorDevice{};
+            if (AzFramework::StringFunc::Equal(multipleDevicesValue.c_str(), "enable"))
+            {
+                AZ_Printf("RHISystem", "\tUsing multiple devices\n");
 
-            for (RHI::Ptr<RHI::PhysicalDevice>& physicalDevice : physicalDevices)
+                usePhysicalDevices = AZStd::move(physicalDevices);
+            }
+            else
```

Comment:
How would you go about deciding, which devices to include?

Reply:
The physical device descriptor already classifies GPUs as either Discrete, Integrated, Virtual, CPU, or Fake (Mock for unit test, etc). I would guess that we probably want options to limit multiGPU to either Only Discrete, All local HW (Discrete + integrated), or All hw (Discrete, Integrated, virtual). We may also need an option to allow multiple Fake GPU's for unit tests.

I'm unaware of a reason we would want to run multiple Software GPU instances, or include a software GPU in the group of GPUS.

I don't think we need this added as a requirement for this change.

We've previously identified that selecting LLVMPipe as the gpu is something we want to avoid (unless explicitly called for) - https://github.com/o3de/o3de/issues/13568